SampleProject/Assets/Scripts/Title.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections;
```

Comment:
[SA1633] The file has no header, the header Xml is invalid, or the header is not located at the top of the file.